tests/integrations/celery/test_celery.py

```diff
@@ -26,9 +26,19 @@ def inner(signal, f):
 
 @pytest.fixture
 def init_celery(sentry_init, request):
-    def inner(propagate_traces=True, backend="always_eager", **kwargs):
+    def inner(
```

Comment:
I updated this `init_celery` to make sure we can also set `monitor_beat_tasks` here.